src/main/java/io/toast/tk/maven/plugin/run/TestPageRunner.java

```diff
@@ -0,0 +1,45 @@
+package io.toast.tk.maven.plugin.run;
+
+import com.google.inject.Module;
+import io.toast.tk.runtime.AbstractScenarioRunner;
+
+public class TestPageRunner extends AbstractScenarioRunner{
+
+	private String reportPath;
+
+	public TestPageRunner(String reportPath, Module[] pluginModules) {
+		super(pluginModules);
+		this.reportPath = reportPath;
+	}
+
+	@Override
+	public String getReportsOutputPath(){
+		return reportPath;
+	}
+
+	@Override
+	public void tearDownEnvironment() {
+		// TODO Auto-generated method stub
+		
+	}
+
+	@Override
+	public void beginTest() {
+		// TODO Auto-generated method stub
+		
+	}
+
+	@Override
+	public void endTest() {
+		// TODO Auto-generated method stub
+		
+	}
+
+	@Override
+	public void initEnvironment() {
+		// TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)